src/selectors/defaultCustomMethods.ts

```diff
@@ -0,0 +1,68 @@
+import {DESCRIPTION_KEY} from '../constants/internal';
+
+import type {GetTestAttributeNameFn, SelectorDefaultCustomMethods} from '../types/internal';
+
+export const createDefaultCustomMethods = (
+  getTestAttributeName: GetTestAttributeNameFn,
+): SelectorDefaultCustomMethods => {
+  const testIdAttrName = getTestAttributeName('id');
+
+  /* eslint-disable sort-keys */
+  return {
+    filterByTestId(testId) {
```

Comment:
`issue`: With the names `TestId`/`TestProp` we would have some discrepancies in the naming of important entities, since in other parts of `e2ed` the names `locatorId`/`locatorProperty` are used for this entities.

Actually I was planning to use the names `locatorId` and `locatorParameter`, and they are already used in `create-locator`, but in `e2ed` I have not yet replaced `locatorProperty` with `locatorParameter`.

After some discussion, we decided, since we are rewriting the selectors, to introduce common names — use `locatorId`/`locatorProperty` everywhere.

No fixes will be needed outside of this pull request. And in it, it seems, it is enough to rename `TestId`/`TestProp` to `LocatorId`/`LocatorProperty` and in some places `property` to `parameter`.